boards/nxp/imx95_evk/imx95_evk-pinctrl.dtsi

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_imx/mimx9596avzxn-pinctrl.dtsi>
+
+&pinctrl {
+
+	lpuart1_default: lpuart1_default {
+		group0 {
+			pinmux = <&iomuxc_uart1_rxd_lpuart_rx_lpuart1_rx>,
+				<&iomuxc_uart1_txd_lpuart_tx_lpuart1_tx>;
+			bias-pull-up;
+			slew-rate = "slightly_fast";
```

Comment:
What's the reason for using different slew rate/drive strength values for this UART (compared to `lpuart3`)?

Reply:
Oh, that's added by me and Zhiqiang Seperately.
Let me change them all to default slightly_fast and x4 since no specific requirement.
Thanks.

Reply:
Checked. The configuration is same with linux dts.